src/api.jl

```diff
@@ -161,13 +182,14 @@ function infer(fieldids::Vector{Tuple{Int, Int, Int}},
 
     # If there are no objects of interest, return early.
     if length(target_sources) == 0
-        return Dict{Int, Dict}()
+        return Dict{Int, Dict}(), (0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0)
```

Comment:
Some additional time measurements are made in ``bin/celeste.jl`` and they are all output there. Also, I want to put ``nputs`` and other such stuff that is used in both places somewhere... putting stuff in ``src/api.jl`` makes it necessary to add the ``Celeste.`` prefix, or to explicitly export things. :(